plugins/uefi/meson.build

```diff
@@ -3,6 +3,11 @@ subdir('efi')
 cargs = ['-DG_LOG_DOMAIN="FuPluginUefi"']
 cargs += '-DEFI_APP_LOCATION_BUILD="' + app.full_path() + '"'
 
+efi_osdir = get_option('efi-osdir')
+if efi_osdir != ''
+  cargs += '-DEFI_OS_DIR="' + efi_osdir + '"'
```

Comment:
it's it a bit weird having `efi_osdir` and `EFI_OS_DIR`? I think either both with underscores before DIR, or neither. I don't mind which :)

Reply:
OK, will make them aligned.